### PR TITLE
Fc 1454 negotiable quote temp shipping address

### PR DESCRIPTION
**Pull request by mGGk-fr**
Adding documentation about temporary shipping address for negotiable quote

[New section](https://deploy-preview-705--heuristic-almeida-1a1f35.netlify.app/docs/2.x/advanced/checkout/temporary-shipping-address#nagociable-quote-checkout)

### CODE REVIEW

File: versioned_docs/version-2.x/advanced/checkout/temporary-shipping-address.mdx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    props.checkoutTransaction(() => {
+      props.setShippingMethod(rest);
+      props.setShippingAdditionalInfo(additionalData || {});
+    });
+  }}
+  onChangeShippingAddress={() => props.gotoStepNumber(0)}
+/>
+```
+
+## Nagociable quote checkout
```
Comment (Kagaminara) [inline versioned_docs/version-2.x/advanced/checkout/temporary-shipping-address.mdx:140]:
```suggestion
## Negotiable quote checkout
```

File: versioned_docs/version-2.x/advanced/checkout/temporary-shipping-address.mdx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+You need to override the `EnhanceAddress` enhancer. And on set checkout shipping
+address, you must add an input to the mutation to specify if you need to save or
+not the address. It's original location is :
```
Comment (Kagaminara) [inline versioned_docs/version-2.x/advanced/checkout/temporary-shipping-address.mdx:144]: **nit:** I find the sentence a bit harsh to read.

**suggestion:** Rework the sentence.
```suggestion
You will need to override the `EnhanceAddress` enhancer, and add an input to the mutation to specify if the address needs to be save or not. Its original location is :
```

Comment (Kagaminara) [inline versioned_docs/version-2.x/advanced/checkout/temporary-shipping-address.mdx:144]:
```suggestion
not the address. Its original location is :
```

**Review by real34 — APPROVED** [review]
Good for me after fixes from @Kagaminara's suggestion are taken into account.

Also, for "git pureness" it'd be better to:
1. merge #702 in `main`
2. merge `main` in `next`
3. rebase this commit so that it only contains changes related to negotiable quotes
… but it's not a big deal :angel:

**Pull request merged**